base/utf16.jl

```diff
@@ -110,7 +110,9 @@ function is_valid_utf16(data::AbstractArray{UInt16})
     return i > n || !utf16_is_surrogate(data[i])
 end
 
-is_valid_utf16(s::UTF16String) = is_valid_utf16(s.data)
+isvalid(str::UTF16String) = isvalid(UTF16String, str.data)
+is_valid_utf16(str::UTF16String) = isvalid(UTF16String, str.data)
+is_valid_utf16(data::AbstractArray{UInt16}) = isvalid(UTF16String, data)
```

Comment:
Yes... if they weren't my own changes, I think I'd be a very good person for reviewing Unicode/strings changes :grinning:  (I spent 20 years being the gatekeeper to the core code at ISC, so I know what's involved!)